source/_posts/2017-05-19 kanban-method

```diff
@@ -0,0 +1,74 @@
+# Kanban method in a consulting company
+
+
+
+Before we start we need to know what is that Kanban. Let me show you a short explanation.
+
+> **Kanban** is a method for managing knowledge work which balances demands for work with the available capacity for new work. 
+Work items are visualized to give participants a view of progress and process, from task definition to customer delivery. 
+Team members "pull" work as capacity permits, rather than work being "pushed" into the process when requested.
+>
+> In software development, Kanban provides a visual process-management system which aids decision-making about what, 
+when and how much to produce. Although the method (inspired by the Toyota Production System and lean manufacturing originated in 
+software development and IT, it may be applied to any professional service whose work outcome is intangible rather than physical.
+
+## Tools
+
+In our company we're using [Trello](https://trello.com/) that have electrical boards to visualize our workflow in all of project. 
+Our arrays are divided into appropriate categories: Backlog, To Do, In Progress, Verification, Done and Project Status, 
+where We can discuss about project features or problems.
+
+![1](http://imageshack.com/a/img922/6156/YfxSE1.png)
+
+## Cards (Kanbans)
+
+According the Kanban method we give out a kanban cards to assigned employees. The card is very important in the whole project process. 
+According to her, we measure our work and set time for each stage in the design, 
+develop and testing solutions in embedded systems projects. Additionally inside the card we can discuss about feature and 
+look what happens last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+
+The card life begins in ToDo, sometimes it happens there we have a lot of big task and cards with small priority are move for Backlog, 
+where they waiting for free moment to solve them. We are trying to minimize transfer cards from right to left, 
```

Comment:
* `they waiting...` -> `they are waiting for some free time`
* `minimize transfer` -> `minimize transferring the cards`